SplunkRumWorkspace/SplunkRum/SplunkRum/AppLifecycle.swift

```diff
@@ -19,6 +19,19 @@ import Foundation
 import UIKit
 import OpenTelemetrySdk
 
+let INACTIVITY_SESSION_TIMEOUT_SECONDS = 15 * 60
+var IS_NEW_SESSION_ID_FOR_INACTIVITY = false
```

Comment:
I'd much rather see a method like `forceNewSessionId` or similar in `Session.swift` being called from the app lifecycle than this global variable that controls what another file's logic does.

Reply:
What do you think of this?  I added an optional parameter to the getRumSessionId function.  When true gets passed it will run generate a new session id.  If not it will follow the normal path.

Reply:
I like that even better.